add-on/src/lib/dnslink.js

```diff
@@ -139,29 +140,29 @@ export default function createDnslinkResolver (getState) {
       // TODO: revisit after https://github.com/ipfs/js-ipfs-api/issues/501 is addressed
       // TODO: consider worst-case-scenario fallback to https://developers.google.com/speed/public-dns/docs/dns-over-https
       const apiCall = `${apiProvider}api/v0/name/resolve/${fqdn}?r=false`
-      const xhr = new XMLHttpRequest() // older XHR API us used because window.fetch appends Origin which causes error 403 in go-ipfs
-      // synchronous mode with small timeout
-      // (it is okay, because we do it only once, then it is cached and read via readAndCacheDnslink)
-      xhr.open('GET', apiCall, false)
-      xhr.setRequestHeader('Accept', 'application/json')
-      xhr.send(null)
-      if (xhr.status === 200) {
-        const dnslink = JSON.parse(xhr.responseText).Path
-        // console.log('readDnslinkFromTxtRecord', readDnslinkFromTxtRecord)
+      const response = await fetch(apiCall, {
+        method: 'GET',
+        headers: {
+          Accept: 'application/json'
+        }
+      })
```

Comment:
do we need to set `ipfs config --json API.HTTPHeaders.Access-Control-Allow-Origin` or other kubo config in any special way for this to work? is the documentation updated with that info?

Reply:
That's right. thanks